pyeudiw/openid4vp/__init__.py

```diff
@@ -5,14 +5,16 @@
 from pyeudiw.sd_jwt import verify_sd_jwt
 
 
-def check_vp_token(vp_token: str, config: dict, sd_specification: dict, sd_jwt: dict) -> Tuple[str | None, dict]:
+def check_vp_token(vp_token: str, config: dict, sd_specification: dict, issuer_jwk: JWK, sd_jwt: dict = {"no_randomness": True}) -> Tuple[str | None, dict]:
```

Comment:
`sd_jwt: dict = {"no_randomness": True}`

please rename `sd_jwt` with something semantically more correct
I read functional parameters not an sd_jwt